service/src/main/java/tds/config/repositories/impl/TimeLimitConfigurationRepositoryImpl.java

```diff
@@ -112,4 +115,21 @@ public TimeLimitConfigurationRepositoryImpl(final NamedParameterJdbcTemplate jdb
 
         return maybeTimeLimitConfig;
     }
+
+    private static class TimeLimitsRowMapper implements RowMapper<TimeLimitConfiguration> {
```

Comment:
I pulled this class in rather than having it as a separate class.  I think we settled on this style after this was originally written.  The added benefit is I can see the names of the columns being mapped rather than jumping back and forth between classes.